src/client.c

```diff
@@ -4,6 +4,7 @@
  * Copyright (C) 2009-2015 Keldon Jones
  *
  * Source file modified by B. Nordli, August 2015.
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
done. While doing this, I added expansion macros in rftg.h, I'll have to go around the code to see where they may be useful.
/*
 * Expansions
 */
#define EXP_BASE    0
#define EXP_TGS     1
#define EXP_RVI     2
#define EXP_BOW     3
#define EXP_AA      4
#define EXP_XI      5